modules/rosa-configure.adoc

```diff
@@ -4,16 +4,16 @@
 // * rosa_cli/rosa-get-started-cli.adoc
 
 [id="rosa-configure_{context}"]
-= Configuring the rosa CLI
+= Configuring the {product-title} (ROSA) CLI (`rosa`)
```

Comment:
As above, makes the title too long - just "ROSA CLI" here I think, as it is specific to that tool